docs/download/downloading-data.md

```diff
@@ -257,3 +257,74 @@ On AWS VM:
 ```
 bin/score-client --profile aws download --manifest manifest.aws-virginia.1556307429042.tsv --output-dir aws_downloads
 ```
+
+
+## Downloading Data from the Azure repository
+
+Microsft cloud service containing ICGC data.
+
+Repository information can be found [here](/download/repositories/#azure)
+
+
+#### Prerequisites
+
+1. [Apply for DACO access](/download/data-access#apply-for-access-to-controlled-data). 
+
+2. Log into your Microsoft Azure account and set up an Azure instance.
+
+3. Download and install score-client software on the Azure VM. Score-client download instructions [here](/download/guide/#installation-of-the-score-client)
+  
+ <span style="color:blue">NOTE: In order to download data from Azure, the score-client tool must run in the same environment as the object storage system. That means you will need your own Azure account to provision a running Azure instance and the Score Client must run on an Azure VM.</span>
+
+
+#### Download Data Instructions
+
+1.  Once your DACO application has been approved, use your OpenID (ie. the Gmail email address you specified in your DACO application) to log into [https://dcc.icgc.org/](https://dcc.icgc.org)
+
+    ![Portal-Login](images/Portal_login.png)
+
+2.  After successful authentication, you will see a green shield icon if you are approved for DACO access, and a green cloud icon if you are approved for Cloud access to the controlled tier. Please note, if you intend to access controlled data in the Cloud, please ensure you complete the "Cloud Storage Access Agreement" section of your DACO application. Otherwise, you will see a red cloud icon.
+
+    ![DACO-Cloud-Access](images/daco-cloud-access.png)
+
+3.  Click on `Token Manager`
+
+    ![Token-Manager-Link](images/token-manager-link.png)
+
+4.  Select `azure.read` and click `Generate` button to create an access token for downloading data from Collaboratory
+
+    ![Token-Manager](images/token-manager-azure.png)
+
+
+5\. Copy and paste this token into your config file for score-client. Click [here](/download/guide/#access-configuration) for instructions on how to configure other elements of score-client. The configuration of the Score Client is stored in the `conf/application.properties` file of the distribution. Edit `application.properties` file to add the generated acesss token. 
```

Comment:
I remember adding this in a while back. It had to do with indentation - the text and figure below it wouldn't line up with each other if I didn't include the `\` (it would look like this otherwise):
![indentation_off](https://user-images.githubusercontent.com/4992044/140830171-392dda0e-0d2f-4dc5-a88e-86470121e53d.png)

Reply:
Oh yes, I see that. Sounds good!